packages/universal-nav/src/webcomponent/Nav.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { colorsPink, layout } from '@pluralsight/ps-design-system-core'
+import { Navbar } from './registry.js'
+function Filler(props) {
+  return (
+    <div
+      slot={props.slot}
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        color: colorsPink.base,
+        height: '100%',
+        border: `2px dashed ${colorsPink.base}`,
+        padding: `0 ${layout.spacingMedium}`
+      }}
+    >
+      {props.children}
+    </div>
+  )
+}
+
+Filler.propTypes = {
+  children: PropTypes.node,
+  slot: PropTypes.string
+}
+export const Nav = () => (
+  <Navbar>
```

Comment:
Got the react component to export properly from the registry without throwing any errors.

Reply:
You could inline style, add classNames or data-attributes on this to manipulate internal styles in shadowDOM that are observing these attributes.